testing/references/bestest_air/kpis_initialize_initial.csv

```diff
@@ -1,7 +1,10 @@
 keys,value
+tdis_tot,0.0
+idis_tot,0.0
 ener_tot,0.0
+cost_tot,0.0
 emis_tot,0.0
-idis_tot,0.0
-tdis_tot,0.0
+pele_tot,4.647544789502356e-36
```

Comment:
Any idea why is this happening?

Reply:
I was imagining something like that, but this explains better. Thanks! I think we can live with such a small numerical diffusion.

Reply:
Yes, I think for now it is fine.